docs/reference/connectors/_category_.yaml

```diff
@@ -1 +0,0 @@
-position: 1
```

Comment:
I think this file is what makes docusaurus display the stuff nicely in the left bar. Currently it adds an `index` entry and doesn't take `index.md` as what should be displayed at the top-level.

Reply:
Nope, this file just moved to now be part of the docs, the reason is the headers.